packages/xchain-terra/package.json

```diff
@@ -45,7 +45,7 @@
   },
   "peerDependencies": {
     "@xchainjs/xchain-client": "^0.11.1",
-    "@xchainjs/xchain-util": "^0.5.1",
+    "@xchainjs/xchain-util": "^0.6.2",
```

Comment:
Latest `@xchainjs/xchain-util` is 0.7.0` now - pls update it in `dev|peerDependencies` for all packages.